plugins/samson_ledger/lib/samson_ledger/client.rb

```diff
@@ -34,17 +34,42 @@ def post_event(deploy)
 
       def build_event(deploy)
         event = {
-          id:           deploy.id,
-          name:         deploy.project.name,
-          actor:        deploy.user.name,
-          status:       deploy.status,
-          started_at:   deploy.created_at.iso8601,
-          summary:      deploy.summary,
-          environment:  deploy.stage.deploy_groups.map(&:environment).uniq.map(&:permalink).map(&:downcase).join(","),
-          url:          deploy.url,
-          pods:         deploy.stage.deploy_groups.map(&:env_value)
+          id:            deploy.id,
+          name:          deploy.project.name,
+          actor:         deploy.user.name,
+          status:        deploy.status,
+          started_at:    deploy.updated_at.iso8601,
+          summary:       deploy.summary,
+          environment:   deploy.stage.deploy_groups.map(&:environment).uniq.map(&:permalink).map(&:downcase).join(","),
+          url:           deploy.url,
+          pods:          pods(deploy.stage.deploy_groups),
+          pull_requests: pull_requests(deploy.changeset)
         }
-        { "events": [event] }
+        {"events": [event]}
+      end
+
+      def pods(deploy_groups)
+        deploy_groups.map { |dg| dg.env_value.match(/^pod(\d+).*/) }.compact.map { |m| m[1].to_i }
+      end
+
+      def pull_requests(changeset)
```

Comment:
Yeah.. But I think that will make things more flexible though. I use the `sanitize-html` library (https://github.com/punkave/sanitize-html) to sanitize the html before rendering. So it "should" be fine (famous last words). 😅